hikari/impl/rest.py

```diff
@@ -214,6 +214,132 @@ def invalidate(self, token: typing.Optional[str]) -> None:
             self._token = None
 
 
+class UserCredentialsStrategy(rest_api.TokenStrategy):
+    """Strategy class for handling client credential OAuth2 authorization.
```

Comment:
Forgot to add it to the review, but this information is also wrong

Reply:
Change client to user or is more than that incorrect

Reply:
Client is not really appropriate here. Should just be OAuth